README.md

```diff
@@ -1,82 +1,68 @@
-django_cinema/README.md¶
-
-                               =============
-                               Django Cinema
-                               =============
-
+README.md
+***
+#DjangoCinema
 Website of Cinema Library written in Django.
+***
+####Include:
 
-Include:
-    - Python
-    - Django
-    - PostgreSQL
-    - Celery
-    - Redis
-    - Django Rest Framework
-    - Gunicorn
-    - Nginx
-    - Docker
-
-Database diagrams:
-    - db_diagram/short_diagram.svg
-    - db_diagram/full_diagram.svg
-
+- Python
+- Django
+- PostgreSQL
+- Celery
+- Redis
+- Django Rest Framework
+- Gunicorn
+- Nginx
+- Docker
+***
+####Database diagrams:
 
-                               -----------
-                               Quick start
-                               -----------
-
-                               DEVELOPMENT
-
-To run current project locally you need:
+- db_diagram/short_diagram.svg
+- db_diagram/full_diagram.svg
+***
+#Quick start
+###I. Development
+####To run current project locally you need:
 
 1. Requirements:
-   - '.env.dev' file with environment variables;
-   - '.env.dev.db' file with DB environment variables;
-2. Start by cloning down this project from GitHub repository.
-3. Set up Python development environment.
-4. Install Docker and Docker-compose.
+    - '.env.dev' file with environment variables
+    - '.env.dev.db' file with DB environment variables
+2. Start by cloning down this project from ***GitHub*** repository.
+3. Set up ***Python*** development environment.
+4. Install ***Docker*** / ***Docker-compose***.
```

Comment:
Why do u use thee astricts symbols here? Is it a new markdown specification?

Reply:
https://texterra.ru/blog/ischerpyvayushchaya-shpargalka-po-sintaksisu-razmetki-markdown-na-zametku-avtoram-veb-razrabotchikam.html